packages/core/src/hierarchy.ts

```diff
@@ -1,29 +1,28 @@
 //
 // Copyright © 2020, 2021 Anticrm Platform Contributors.
-// 
+//
 // Licensed under the Eclipse Public License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License. You may
 // obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
-// 
+//
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// 
+//
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
 
 import type { Ref, Class, Obj, Data, Domain } from './classes'
-import type { Tx, TxCreateObject } from './tx'
+import type { Tx, TxCreateDoc } from './tx'
 
 import core from './component'
 
 export class Hierarchy {
-
   private readonly classes = new Map<Ref<Class<Obj>>, Data<Class<Obj>>>()
 
-  getAncestors(_class: Ref<Class<Obj>>): Ref<Class<Obj>>[] {
-    const result: Ref<Class<Obj>>[] = []
+  getAncestors (_class: Ref<Class<Obj>>): Array<Ref<Class<Obj>>> {
+    const result: Array<Ref<Class<Obj>>> = []
```

Comment:
Why? :)

Reply:
Complex types by standard rule require Array to be used.

Reply:
Same shit, I'm pretty sure `ts-standard` does not force this rule. Also and thechnically `Ref`s are strings so this rule looks dumb for refs. Can we explore differences with `ts-standard`?

Reply:
ts-standard output
```
 core git:(object-doc-eslint) ✗ ts-standard src
ts-standard: Typescript Standard Style! (https://github.com/standard/ts-standard)
ts-standard: Run `ts-standard --fix` to automatically fix some problems.
  /Users/haiodo/Develop/private/hardware/platform/packages/core/src/hierarchy.ts:25:19: Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead. (@typescript-eslint/array-type)
```

Reply:
ts-standard --fix automatically wrap it with Array.